### ISSUE TITLE
Native binding path

### ISSUE THREAD

**Issue by anandthakker** [naturalatlas/node-gdal#152]
I'm very unfamiliar with the guts of native node bindings, so apologies if this question is off the mark.

**tl;dr:** this module's binding seems to end up in `node_modules/gdal/lib/binding/gdal.node`, whereas others, e.g. node-sqlite3, go into a directory describing the version/platform/architecture, like: `node_modules/sqlite3/lib/binding/node-v46-darwin-x64/node_sqlite3.node`.  Why is this?  I dug around a bit in the gyp config files, but didn't really know what I was looking for -- the path-related stuff seems pretty similar in this module and sqlite3.

**Context:** I'm trying to get this to work with `electron-rebuild`, as a postinstall step for an electron app.   I saw #127, but I _think_ the reason it's not working for me is that if I do `npm rebuild gdal --runtime=electron --target=0.36.10 --arch=x86`, the node-pre-gyp step looks for `gdal/lib/binding/gdal.node`, finds it, and so thinks there's nothing to do.  By contrast, when I do the same thing for `sqlite3`, pre-gyp looks for `sqlite3/lib/binding/electron-v0.36-darwin-x64/node_sqlite3.node`, which it _doesn't_ find, and so it actually does rebuild the module.

**Comment by anandthakker** [comment]
> I dug around a bit in the gyp config files, but didn't really know what I was looking for -- the path-related stuff seems pretty similar in this module and sqlite3.

Oh -- just noticed, looks like this is actually set in `package.json`.  Any chance maintainers would take a PR to change the path to something like https://github.com/mapbox/node-sqlite3/blob/master/package.json#L12?

**Comment by brianreavis** [comment]
Hey @anandthakker, sorry for the delay. To answer your question: sure, why not! I see no harm in that. A PR would be great :+1:

**Comment by anandthakker** [comment]
@brianreavis :+1: cool, there ya go ^.  Worked locally for me, but again, haven't done any native binding development myself, so lemme know if it needs changing
